src/OSPSuite.Core/Domain/Builder/ExpressionTypes.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace OSPSuite.Core.Domain.Builder
+{
+
+   public enum ExpressionTypesId
+   {
+      Transport,
+      Enzyme,
+      BindingPartner
+   }
+
+   public static class ExpressionTypes
+   {
+      public static ExpressionType TransportProtein = new ExpressionType(ExpressionTypesId.Transport, Assets.IconNames.Transporter, Assets.Captions.Transporter);
+      public static ExpressionType MetabolizingEnzyme = new ExpressionType(ExpressionTypesId.Enzyme, Assets.IconNames.Enzyme, Assets.Captions.Enzyme);
+      public static ExpressionType ProteinBindingPartner = new ExpressionType(ExpressionTypesId.BindingPartner, Assets.IconNames.Protein, Assets.Captions.Protein);
+
+      private static readonly List<ExpressionType> _types = new List<ExpressionType>
+      {
+         TransportProtein,
+         MetabolizingEnzyme,
+         ProteinBindingPartner
+      };
+
+      public static ExpressionType ById(ExpressionTypesId expressionTypeId)
+      {
+         return _types.First(x => Equals(x.Id, expressionTypeId));
```

Comment:
what about you make it a cache by id here so that you retrieve by id? It seems more idiomatic

Reply:
done